src/track-type-dependency.js

```diff
@@ -30,3 +44,15 @@ export function printTypes() {
   // eslint-disable-next-line
   console.log(trackedTypes().join());
 }
+
+export function trackedFields() {
+  return Object.getOwnPropertyNames(fields).reduce((acc, key) => {
+    acc[key] = Object.getOwnPropertyNames(fields[key]);
+
+    return acc;
+  }, {});
+}
+
+const Tracker = {trackTypeDependency, trackFieldDependency};
```

Comment:
This needs to be a single export because of how we're stripping it to a noop in production, and noop only exports default. Ideally these should be separate modules, but I'm hoping to get rid of `trackTypeDependency` and just have a single `trackDependency` call in the near future.